engine/render/src/render/font_renderer.cpp

```diff
@@ -676,7 +677,7 @@ namespace dmRender
         }
     }
 
-    static int CreateFontVertexDataInternal(TextContext& text_context, HFontMap font_map, const char* text, const TextEntry& te, float recip_w, float recip_h, GlyphVertex* vertices, uint32_t num_vertices)
```

Comment:
Note, i've removed the "const" modifier in ordrer to update radius and center on TextEntry.